certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/citizen/service/GetCitizenCertificateListService.java

```diff
@@ -39,6 +39,10 @@ public GetCitizenCertificateListResponse get(
     return GetCitizenCertificateListResponse.builder()
         .citizenCertificates(certificates.stream()
             .filter(certificate -> certificate.certificateModel().availableForCitizen())
+            .filter(
```

Comment:
Would this belong better as a function of Certificate or Patient object?

Reply:
I think it makes sense to move the logic to the Patient object!

Reply:
Doesn't seem like we've done it for protected person or deceased so maybe it's overkill since it's only a boolean

Reply:
Also doesn't seem like it can be null